src/home/apis/authVerification.ts

```diff
@@ -0,0 +1,48 @@
+import { AxiosError } from 'axios';
+
+import { authClient } from '@/apis';
+import { STORAGE_KEYS } from '@/constants/storage.constant.ts';
+
+import {
+  AuthErrorData,
+  GetAuthVerificationCheckResponse,
+  PostAuthVerificationEmailResponse,
+} from '../types/Auth.type';
+
+interface EmailVerificationParams {
+  email: string;
+  verificationType: 'SIGN_UP' | 'PASSWORD';
+}
+
+interface VerificationCheckParams {
+  session: string;
+}
+
+export const postAuthVerificationEmail = async (
+  emailVerificationParams: EmailVerificationParams
+): Promise<PostAuthVerificationEmailResponse> => {
+  try {
+    const res = await authClient.post(`/auth/verification/email`, emailVerificationParams);
+    if (res.data) {
+      sessionStorage.setItem(STORAGE_KEYS.EMAIL_AUTH_SESSION_TOKEN, res.data.sessionToken);
+      sessionStorage.setItem(
+        STORAGE_KEYS.EMAIL_AUTH_SESSION_TOKEN_EXPIRED_IN,
+        res.data.sessionTokenExpiredIn.toString()
+      );
+    }
+    return { data: res.data };
+  } catch (error: unknown) {
+    return { error: error as AxiosError<AuthErrorData> };
+  }
+};
+
+export const getAuthVerificationCheck = async ({
+  session,
+}: VerificationCheckParams): Promise<GetAuthVerificationCheckResponse> => {
+  try {
+    const res = await authClient.get(`/auth/verification/check?session=${session}`);
+    return { data: res.data };
```

Comment:
다른 API와 통일성을 유지하기 위해 query parameter는 axios에서 제공하는 config 객체에 명시하는게 좋을 것 같습니다!
```suggestion
    const res = await authClient.get('/auth/verification/check', {
      params: {
        session,
      },
    });
```

Reply:
반영했습니당 ([refactor: query parameter config에 명시](https://github.com/yourssu/Soomsil-Web/pull/158/commits/12a30bd7ecf2b0ef05c416c2463014222a77b297))